src/main/java/com/synopsys/integration/alert/Application.java

```diff
@@ -84,72 +68,11 @@ void init() {
         startupManager.startup();
     }
 
-    @Bean
-    public PlatformTransactionManager transactionManager() {
-        return new ResourcelessTransactionManager();
-    }
-
-    @Bean
-    public MapJobRepositoryFactoryBean mapJobRepositoryFactory() throws Exception {
-        final MapJobRepositoryFactoryBean factory = new MapJobRepositoryFactoryBean(transactionManager());
-        factory.afterPropertiesSet();
-
-        return factory;
-    }
-
-    @Bean
-    public JobRepository jobRepository() throws Exception {
-        return mapJobRepositoryFactory().getObject();
-    }
-
-    @Bean
-    public SimpleJobLauncher jobLauncher() {
-        final SimpleJobLauncher launcher = new SimpleJobLauncher();
-        try {
-            launcher.setJobRepository(jobRepository());
-        } catch (final Exception ex) {
-            logger.error("Creating job launcher bean", ex);
-        }
-        return launcher;
-    }
-
-    @Bean
-    public TaskScheduler taskScheduler() {
-        final ThreadPoolTaskScheduler threadPoolTaskScheduler = new ThreadPoolTaskScheduler();
-        threadPoolTaskScheduler.setPoolSize(Runtime.getRuntime().availableProcessors());
-        return threadPoolTaskScheduler;
-    }
-
-    @Bean
-    public TaskExecutor taskExecutor() {
-        return new SyncTaskExecutor();
-    }
-
-    @Bean
-    public Gson gson() {
-        return new GsonBuilder().setDateFormat(RestConstants.JSON_DATE_FORMAT).create();
-    }
-
-    @Bean
-    public AuthenticationSupport authenticationSupport() {
-        return new AuthenticationSupport();
-    }
-
-    @Bean
-    public HttpSessionCsrfTokenRepository csrfTokenRepository() {
-        return new HttpSessionCsrfTokenRepository();
-    }
-
     @Bean
     public DaoAuthenticationProvider alertDatabaseAuthProvider(final PasswordEncoder defaultPasswordEncoder) {
```

Comment:
Do we still want this here? or inApplicationConfiguration?

Reply:
StartupManager cannot be autowired correctly if I try to separate it out.  The StartupManager class has too many dependencies that we need to look into refactoring.  This way the configuration is either in ApplicationConfiguration or AuthenticationHandler